spring-data-rest-webmvc/src/test/java/org/springframework/data/rest/webmvc/RepositoryRestExceptionHandlerUnitTests.java

```diff
@@ -47,7 +47,7 @@ class RepositoryRestExceptionHandlerUnitTests {
 	@BeforeAll
 	public static void silenceLog() {
 
-		logger = (Logger) LoggerFactory.getLogger(RepositoryRestExceptionHandler.class);
+		logger = (Logger) LoggerFactory.getLogger(RepositoryRestExceptionHandlerUnitTests.class);
```

Comment:
This one is not useful as it accomplishes something else. The recipe would benefit from considering tests and where (method or class body) the logger is actually obtained.